package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "amqp-lib",
-    "version": "1.0.26",
+    "name": "amqp-lib-ts",
```

Comment:
Won't this change the name in npm?

Reply:
amqp-lib-ts 
that is the correct name
And it is published in npm as amqp-lib-ts